src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends AbstractFigure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq. units, width: " + width + " units, height: " + height + " units, color: " + getColor());
```

Comment:
Don't use `System.out.println()` in the `draw()` method for figures. Instead, consider defining a `toString()` method that returns the string representation of the object, and print it externally if needed.